### PR TITLE
Complete Portfolio Page 

### PR DESCRIPTION
**Pull request by Aradhya-B**
image

# Single page portfolio 
Adds interactivity using external js libraries: 
- typed.js: typing animation library (https://github.com/mattboldt/typed.js/)
- lightbox.js: used to overlay images on current page (https://lokeshdhakar.com/projects/lightbox2/)
- baffle.js: obfuscates and reveals text in DOM elements (https://camwiegert.github.io/baffle/)
- breathing-halftone.js: makes images go woah with lots of floaty dots (https://breathing-halftone.desandro.com/)

## LightBox
image

## Typed
image

## Baffle
image

## Breathing half tone
image

### CODE REVIEW

File: portfolio/src/main/webapp/styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  font-size: 1.5em;
+}
+
+.baffle-inner {
+  color: #0c8716;
+  text-align: center;
+}
+
+.portrait {
+    display: flex;
```
Comment (cftorres) [inline portfolio/src/main/webapp/styles/styles.css:92]: Nit: indentation here different than all the others

**Review by nickKelman — CHANGES_REQUESTED** [review]
Can you break this pull requests into more granular requests?

It seems like you are doing a lot at once here.

**Comment by Aradhya-B** [comment]
I can do that although I'm not sure if I'll be able to isolate them completely by feature just because the commit history is kind of messy, but I'll try my best! Will close this one then.

**Comment by nickKelman** [comment]
Thanks, Yeah i know we talked about this in our 1:1, I think there is some low hanging fruit here that should be easy to split out such as adding the breathing halftone script or the images in there own pull requests. If it proves to be too much of a headache let me know.

**Pull request closed without merging**